subjects/fillit.en.md

```diff
@@ -0,0 +1,109 @@
+## fiil it
+
+### Objectives
+
+Develop a program that receives only one parameter, a txt file which will contain a list of [**Tetrominos**](https://en.wikipedia.org/wiki/Tetromino) to assemble them in order to create the smallest square possible.
+
+### Instructions
+
+All files referring to the project, must be in the same folder.
+
+The program must:
+
+- Be written in **GO**
+- Be compiled to be runned
+- Assemble all of the **Tetrominos** in order to create the smallest square possible
+- Show an error if the input doesn't correspond to a classic **Tetromino** piece
+- Not allow the rotation of the **Tetrominos**
+- Organize the **Tetrominos** by their apparition order in the file
+- Identify each **Tetromino** in the solution, by assigning different charactes to different **Tetrominos**
+- Have at least 2 **Tetrominos** in the text file
```

Comment:
This list starts with "The program must:"
So I'm reading "The program must have at least 2 **Tetrominos** in the text file" which is inaccurate.

Reply:
What we mean is that text file should have at least two pieces (Tetrominos).

Reply:
so this line should be :
- Expect at least 2 tetrominoes in the text file
so that the student will read : "The program must expect at least 2 tetrominoes in the text file"